src/app/(BottomTabNavigation)/AllCases/CaseScreen/styles.ts

```diff
@@ -38,4 +37,17 @@ export default StyleSheet.create({
     fontWeight: 'bold',
     textAlign: 'center',
   },
+  shareContainer: {
+    justifyContent: 'flex-end',
+    flexDirection: 'row',
+    marginTop: 10,
+    marginLeft: 280,
```

Comment:
Try putting the share container in a larger container and pushing it to the right using `flex-end`. Or, because the main content occupies `92%` of the screen width, I'd even prefer a `marginRight: '4%'` so it adjusts according to screen size.